protocol/x/prices/genesis_test.go

```diff
@@ -9,16 +10,54 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestGenesis(t *testing.T) {
+func TestInitExportGenesis(t *testing.T) {
+	tests := map[string]struct {
+		genesisState *types.GenesisState
+	}{
+		"default genesis": {
+			genesisState: types.DefaultGenesis(),
+		},
+		"empty genesis": {
```

Comment:
nit: how about the case where the the state is updated after init genesis and then exported?